bootstrap.sh

```
@@ -15,6 +15,8 @@ kubectl apply -f .infrastructure/app/nodeport.yml
 kubectl apply -f .infrastructure/app/hpa.yml
 kubectl apply -f .infrastructure/app/deployment.yml
 
+kubectl aooly -f .infrastructure/security/rbac.yml
```

There is a typographical error that will cause the script to fail at this line and therefore the RBAC manifest will not be applied. This blocks the checklist requirement to apply the RBAC manifest (and thus will prevent validators from seeing the ServiceAccount/Role/RoleBinding in-cluster).

Suggestions:
- Replace `kubectl aooly -f .infrastructure/security/rbac.yml` with `kubectl apply -f .infrastructure/security/rbac.yml`.
- Consider adding `set -e` near the top of the script to fail fast on errors, or test the exit code after critical applies and exit with a helpful message.

This fix is required for successful validation of the task (the RBAC resources must be applied).